JeMPI_Apps/JeMPI_UI/jest.config.js

```diff
@@ -0,0 +1,22 @@
+const { pathsToModuleNameMapper } = require('ts-jest')
+const { compilerOptions } = require('./tsconfig')
+
+module.exports = {
```

Comment:
Hi @issambaccouch . Here you, and thanks for the links above. I do think those links have to do more with how jest loads your source code whilst testing (traditionally it used to transpile esm source using transforms before running the tests. See https://jestjs.io/docs/code-transformation). This is just the config file used to run the tests. We could in theory convert it into esm, but this will result in implemeting a few work arounds to make it work (as esm support is still experimental in jest). Also, jest documents still recommends cjs for js config files. See    https://jestjs.io/docs/configuration